source/command-monitoring/tests/unified/redacted-commands.yml

```diff
@@ -25,22 +26,27 @@ tests:
         arguments:
           commandName: authenticate
           command:
-            authenticate: "private"
-        # Malformed authentication commands will fail against the server, but we
-        # just want to assert that security-related commands are redacted
-        # in command monitoring.
+            authenticate: 1
+            mechanism: "MONGODB-X509"
+            user: "CN=myName,OU=myOrgUnit,O=myOrg,L=myLocality,ST=myState,C=myCountry"
+            db: "$external"
+        # An authentication error is expected, but we want to check that the
+        # CommandStartedEvent is redacted
         expectError:
```

Comment:
Is this a case where you might want to use `ignoreResultAndError` (introduced in schema version 1.3)?

Reply:
As discussed in Slack, doing so would complicate integrating these tests into older driver versions for the patch. Fine to keep as-is, since we do know these commands will error anyway.